packages/core/entity/FileDetail/index.ts

```diff
@@ -85,12 +87,69 @@ export default class FileDetail {
         return FileDetail.convertAicsS3PathToHttpUrl(`${AICS_FMS_S3_BUCKET}${pathWithoutDrive}`);
     }
 
+    private static generateFilePath(env: Environment, fileName: string, fmsId: string): string {
```

Comment:
Does generateFilePath have test cov?

Reply:
Not any more.... removed in [3bee712](https://github.com/AllenInstitute/biofile-finder/pull/389/commits/3bee712c637ad2bf7571bed19b960fe11de85598)